epub33/overview/index.html

```diff
@@ -529,10 +531,10 @@ <h2>Navigation</h2>
 					flexible navigation system.</p>
 
 				<p>The Navigation Document can also be reused in the body of an EPUB Publication by including it in the
-						<a data-cite="epub-33#sec-spine-elem"><code>spine</code></a>. To avoid the situation in highly
-					structured documents where it might not be desirable to display the complete table of contents to
-					users in the body, the display level can be modified using the [[HTML]] <a
-						data-cite="html#the-hidden-attribute"><code>hidden</code></a> attribute. This attribute is
+						<a data-cite="epub-33#sec-spine-elem"><code>spine</code></a>. To avoid the situation in 
+						highly-structured documents where it might not be desirable to display the complete table of contents to
+					users in the body of the publication, the display level can be modified using the [[HTML]] <a
+						data-cite="html#sec-nav-doc-use-spine"><code>hidden</code></a> attribute. This attribute is
```

Comment:
Assume you want to change this to an EPUB reference?
```suggestion
					users in the body of the publication, the display level can be modified using the <a
						data-cite="epub-33#sec-nav-doc-use-spine"><code>hidden</code> attribute</a> [[EPUB-33]]. This attribute is
```